Examples/Cpp/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_policy(SET CMP0063 NEW)
```

Comment:
`CMP0063` was introduced in CMake 3.3. You need to change `cmake_minimum_required` in the root `CMakeLists.txt` file.